### PR TITLE
Improve Player movement and collisions

### PR DESCRIPTION
**Pull request by 0xNonCents**
The main goal of this PR is for the player to slide down sloped surfaces and completely fall down vertical walls.

### CODE REVIEW

File: src/lib/components/map/ObstacleObjects.svelte [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
```
Comment (0xNonCents) [inline src/lib/components/map/ObstacleObjects.svelte:1]: This file generates random objects to practice parkour on.

File: static/DrawHere.png
Comment (0xNonCents) [inline static/DrawHere.png:1]: Ignore this change, I accidently made a commit where I changed DrawHere.png

File: src/lib/components/map/SplashWall.svelte [hunk 2]
```diff
@@ -30,7 +30,7 @@
 </script>
 
 <T.Group position={[0, 27, -50]}>
-        <AutoColliders shape={'cuboid'}>
+        <AutoColliders shape={'cuboid'} friction={0}>
```
Comment (0xNonCents) [inline src/lib/components/map/SplashWall.svelte:33]: The key to not sticking to the wall is setting a low friction.

**Comment by 0xMugen** [comment]
Can you have a state in the stores canPaint for me to identify if the user is alowed to go into paint mode or not

**Review by 0xMugen — APPROVED** [review]

**Pull request merged**